src/Controllers/LoginController.php

```diff
@@ -55,10 +63,29 @@ public function loginAction()
         return redirect()->to(config('Auth')->loginRedirect())->withCookies();
     }
 
+    /**
+     * Returns the rules that should be used for validation.
+     *
+     * @return string[]
+     */
+    protected function getValidationRules(): array
+    {
+        $rules = [
+            'email'    => 'required',
+            'password' => 'required',
+        ];
+
+        if (setting('Auth.validFields') === ['email']) {
+            $rules['email'] .= '|valid_email';
```

Comment:
Why do you add this only when `setting('Auth.validFields') === ['email']`?

Reply:
I was watching "myth-auth"

https://github.com/lonnieezell/myth-auth/blob/bc606fed47812773bb0647c7770fae6edc02d9f4/src/Controllers/AuthController.php#L71-L73